### ISSUE TITLE
Suggestion: Allow creating new repos

### ISSUE THREAD

**Issue by vijayant123** [slapperwan/gh4a#369]
Maybe we can add a button in the sidebar menu to create new repositories on the user's GitHub account.

I would love to implement this if you would accept the pull request.

**Comment by Tunous** [comment]
Perhaps instead place it in the toolbar menu when viewing "My Repositories" screen.

**Comment by maniac103** [comment]
Yeah, that would make sense, but what I'm not convinced on yet is the use case: why would on want to create an (empty) repo on mobile when one can't push code there anyway at that moment?

**Comment by Tunous** [comment]
Indeed. I didn't think of that...

**Comment by vijayant123** [comment]
I felt the need for this when browsing the activity feed on the App. 

We should be able to create an empty repo on the mobile app and then able to push code from a terminal where we don't have access to the browser.

Here is what I usually do:

1. In my off hours, I am browsing the activity feed and see an interesting repository or get an idea for a project.
2. Since I am not at my workstation, I want to be able to at least create an empty repository from the app itself, so that I can work on it the next day and that idea gets to be listed in my repositories. This way I don't loose that idea.
3. It acts as a todo list.

**Comment by smith901** [comment]
Would be nice. I need to make a quick repo but can't since I'm nowhere near my computer.
The repo is needed since I can't add an image to an issue using OctoDroid. I don't want to sign up to yet another service to do something simple.
@Tunous I looked for the capability in that exact spot. Its an intuitive place anyone would look.
I see @maniac103 says we can't push code to the repo, so I guess I'm stuck trying until I get to my unit. This feature might not be useful to the devs but it is useful to some of us. Kinda strange this app can do what it does but pushing code to one's repo isn't an ability. We already logged on.

Dropped this here to not create another issue since its Open.
